Reflection.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Created by Ericka-VS on 23/07/2016.
+ */
+
+import java.lang.reflect.*;
+import java.util.ArrayList;
+
+public class Reflection {
+
+    // public String nameClass;
+    private Class classObject;
+
+    /**
+     * Constructor that receive a name String (of a class)
+     * were is created an object class with the name received
+     *
+     * @param className
+     * @throws ClassNotFoundException
+     */
+    public Reflection(String className) throws ClassNotFoundException {
```

Comment:
The reflection class should handle the exception, and not be thrown until your main class